dcrdtest/rpc_harness_test.go

```diff
@@ -788,3 +794,37 @@ func TestSetupWithWrongDcrd(t *testing.T) {
 		t.Fatalf("Unexpected error during TearDown: %v", err)
 	}
 }
+
+// TestKeepsNodeDir asserts that when the harness is set to keep the node dir,
+// it is not removed after TearDown().
+func TestKeepsNodeDir(t *testing.T) {
```

Comment:
`TestKeepNodeDir`?  Either is fine imo, but it's more common to name them `Test<Method>`.